packages/warehouses/src/warehouseClients/BigqueryWarehouseClient.ts

```diff
@@ -528,7 +528,7 @@ export class BigqueryWarehouseClient extends WarehouseBaseClient<CreateBigqueryC
             )}`,
         );
         return new WarehouseQueryError(
-            `Bigquery warehouse error: ${error?.reason}`,
+            `Bigquery warehouse error: ${error?.reason} - ${error?.message}`,
```

Comment:
The `error.reason` field is often too generic to determine the root cause of an error. For example, if Location is mistyped, the reason is simply `invalid`, which lacks context. To aid in debugging, I've added the more detailed `error.message` to the logs.